From 45f3d98b9b4237e6787bff101535b0d8bcd5ac67 Mon Sep 17 00:00:00 2001
From: Stephane Danieau <stephane.danieau@st.com>
Date: Wed, 7 Mar 2018 10:45:17 +0100
Subject: [PATCH 5/6] drm: Add DRM_MODE_FB_YFCR flag definition

    For a YUV buffer, this "YUV Full Color Range" flag specifies
    components color range.
    If set, components are in full color range.
    If not set, components are in limited color range.

Signed-off-by: Stephane Danieau <stephane.danieau@st.com>

diff --git a/include/drm/drm_mode.h b/include/drm/drm_mode.h
index 2f2208b..03d40e6 100644
--- a/include/drm/drm_mode.h
+++ b/include/drm/drm_mode.h
@@ -386,6 +386,8 @@ struct drm_mode_fb_cmd {
 #define DRM_MODE_FB_INTERLACED	(1<<0) /* for interlaced framebuffers */
 #define DRM_MODE_FB_MODIFIERS	(1<<1) /* enables ->modifer[] */
 #define DRM_MODE_FB_BFF		(1<<2) /* if interlaced, bottom field first */
+#define DRM_MODE_FB_YFCR		(1<<3) /* if YUV full color range */
+
 
 struct drm_mode_fb_cmd2 {
 	__u32 fb_id;
-- 
2.7.4

